src/markdown_generators.js

```diff
@@ -226,6 +228,7 @@ const generateCourseSectionFrontMatter = (
     Generate the front matter metadata for a course section given a title and menu index
     */
   const courseSectionFrontMatter = {
+    uid:       pageId,
```

Comment:
This is value is `course_home` for `res-2-006-girls-who-build-cameras-summer-2016`. Is that the right value for this?

Reply:
I'll need to look into this tomorrow, but I think there might be a data issue with some of the courses.  That value is a default in the event the course home page is not found for whatever reason.  Stay tuned.

Reply:
@noisecapella I spent some time looking into this today, and digging through the Plone JSON exports for that course I didn't find any pages of the type `CourseHomeSection`.  It doesn't seem to have a "course home" in the original data, in which case the default of `course_home` makes sense for now I think.

Reply:
Actually, I just found that for "resource" courses like this, the type seems to be `SRHomePage`.  This type will have to be added to the types parsed by `ocw-data-parser` and added to the `course_pages` array.  We can either merge this as-is for now, or hold off until I can push through an `ocw-data-parser` PR to pull in those other page types.

Reply:
@noisecapella PR here for that: https://github.com/mitodl/ocw-data-parser/pull/117